cpp/oneapi/dal/algo/svm/backend/cpu/train_cls_kernel.cpp

```diff
@@ -165,10 +164,10 @@ static result_t call_binary_daal_kernel(const context_cpu& ctx,
     daal_svm_parameter.cacheSize = cache_byte;
     daal_svm_parameter.svmType = daal_svm::training::internal::SvmType::classification;
 
-    auto arr_label = row_accessor<const Float>{ labels }.pull();
     binary_label_t<Float> unique_label;
-    auto arr_new_label = convert_labels(arr_label, { Float(-1.0), Float(1.0) }, unique_label);
-    const auto daal_labels = interop::convert_to_daal_homogen_table(arr_new_label, row_count, 1);
+    const auto new_labels =
+        convert_binary_labels(labels, { Float(-1.0), Float(1.0) }, unique_label);
```

Comment:
`unique_label` is not a clear name in this context, because the previous parameter does also correspond to unique labels.
Maybe `old_unique_labels`?